GoogleUtilities/Environment/GULHeartbeatDateStorageUserDefaults.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithDefaults:(NSUserDefaults *)defaults key:(NSString *)key
   return self;
 }
 
-- (nullable NSMutableDictionary *)heartbeatDictionaryFromDefaults {
+- (NSMutableDictionary *)heartbeatDictionaryFromDefaults {
```

Comment:
Removing `nullable` attribute to return type because a non-nil NSMutableDictionary will always be returned.